diff --git a/node_modules/react-native-disable-battery-optimizations-android/android/build.gradle b/node_modules/react-native-disable-battery-optimizations-android/android/build.gradle
index a28f0dd..59d8ab5 100644
--- a/node_modules/react-native-disable-battery-optimizations-android/android/build.gradle
+++ b/node_modules/react-native-disable-battery-optimizations-android/android/build.gradle
@@ -31,6 +31,6 @@ repositories {
 }
 
 dependencies {
-    compile 'com.facebook.react:react-native:+'
+    implementation 'com.facebook.react:react-native:+'
 }
   
\ No newline at end of file
